hypervisor/arch/x86/cpu.c

```diff
@@ -267,7 +267,7 @@ static uint16_t get_pcpu_id_from_lapic_id(uint32_t lapic_id)
 	uint16_t i;
 	uint16_t pcpu_id = INVALID_CPU_ID;
 
-	for (i = 0U; (i < phys_cpu_num) && (i < CONFIG_MAX_PCPU_NUM); i++) {
+	for (i = 0U; i < phys_cpu_num; i++) {
```

Comment:
add @pre for phys_cpu_num

and we need to add @post for get_pcpu_nums()

Reply:
ok will add @pre for phys_cpu_num
 add @post for get_pcpu_nums()